tools/gromacs/trj.xml

```diff
@@ -140,23 +154,27 @@
                 </conditional>
             </when>
             <when value="trjcat">
-                <param name="trj_input" type="data" format='xtc,trr' label="Input trajectories" help="In XTC or TRR format, but please do not mix the two. Please note this tool does not currently take order into account when concatenating." multiple="true"/>
+                <param name="trj_input" type="data" format="xtc,trr" label="Input trajectories" help="In XTC or TRR format, but please do not mix the two. Plese note that the order of concatation is the same as the consecutive order of these selected inputs." multiple="true"/>
```

Comment:
how did you fix this?

Probably you should distinguish these two params, if it is really necessary to have both.

Reply:
this is so the command line is executed the same under trjcat, whether the user is just selecting that single multiple select input, or using the repeats. this was important because this ensures backwards compatibility with workflows using the older versions that only would have used the single input.